.github/workflows/pr_push.yml

```diff
@@ -48,6 +48,31 @@ jobs:
         - name: Build
           run: >
             cmake --build ${{github.workspace}}/build --config Release -j ${{matrix.nproc}}
+
+    CodeStyle:
+      name: Coding style
+      runs-on: ubuntu-latest
+
+      steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install apt packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake clang-format-15 libjemalloc-dev libnuma-dev libtbb-dev
```

Comment:
Libs are redundant as you don't compile the sources, just disable building parts of umf which require these libs.

Reply:
Done